Jenkinsfile

```diff
@@ -62,6 +63,12 @@ pipeline {
                         expression { params.unit_validate }
                     }
                     agent { label 'amd64 && ubuntu-1804 && overlay2' }
+                    environment {
+                        // On master ("non-pull-request"), force running some validation checks (vendor, swagger),
+                        // even if no files were changed. This allows catching problems caused by pull-requests
+                        // that were merged out-of-sequence.
+                        TEST_FORCE_VALIDATE = sh returnStdout: true, script: 'if [ "${BRANCH_NAME%%-*}" != "PR" ] || [ "${CHANGE_TARGET:=master}" != "master" ] || [ "${validate_force}" = "true" ]; then echo "1"; fi'
```

Comment:
I think `${CHANGE_TARGET:=master}` here was intended to be `${CHANGE_TARGET:-master}`? (given the script's length it really doesn't make much difference, but IMO it's better to make sure the implementation matches the intention without side-effect :sweat_smile:)